compiler/rustc_mir_build/src/check_unsafety.rs

```diff
@@ -503,17 +503,11 @@ impl<'a, 'tcx> Visitor<'a, 'tcx> for UnsafetyVisitor<'a, 'tcx> {
                 // THIR desugars UNSAFE_STATIC into *UNSAFE_STATIC_REF, where
                 // UNSAFE_STATIC_REF holds the addr of the UNSAFE_STATIC, so: take two steps
```

Comment:
Trimmed?

Reply:
Ah, I need to update this comment. We don't care about statics here anymore.